rsocket-core/src/main/java/io/rsocket/plugins/DuplexConnectionInterceptor.java

```diff
@@ -23,7 +23,7 @@
 public @FunctionalInterface interface DuplexConnectionInterceptor
     extends BiFunction<DuplexConnectionInterceptor.Type, DuplexConnection, DuplexConnection> {
   enum Type {
-    STREAM_ZERO,
+    SETUP,
```

Comment:
Why change this from STREAM_ZERO which is anything related to the connection, to setup?

Reply:
Because interceptors are applied in `ClientServerInputMultiplexer`, and It considers SETUP frame only as part of STREAM_ZERO connection - https://github.com/rsocket/rsocket-java/blob/e48658e28676b6dc6dcef20b2fb70b38f220ad8d/rsocket-core/src/main/java/io/rsocket/internal/ClientServerInputMultiplexer.java#L77.